shacl12-profiling/index.html

```diff
@@ -511,151 +490,671 @@ <h3>Document Conventions</h3>
+    </p>
+    <p>
+      The syntactic rules specified here are given in English and use keywords MAY, MUST, MUST NOT, RECOMMENDED, SHOULD, and SHOULD NOT that are to be interpreted as described in <a href="https://www.rfc-editor.org/info/bcp14">BCP 14</a> [[RFC2119]] [[RFC8174]] when, and only when, they appear in all capitals, as shown here.
+    </p>
+    <p>
+      Conformance claims of data to this specification's rules can be tested by validation using the SHACL-SHACL Profiling validator in the <a href="https://w3c.github.io/data-shapes/shacl12-overview/#shacl-shacl">SHACL-SHACL Annex</a> of the SHACL Overview specification <em>[<a href="https://w3c.github.io/data-shapes/shacl12-overview/">shacl12-overview</a>]</em>
+    </p>
+  </section>
+</section>
+
+<section id="packaging">
+  <h2>Packaging SHACL</h2>
+  <p>
+    Packaging of SHACL is the presentation of SHACL elements - shapes, rules, etc. - in relation to container objects.
+  </p>
+  <section id="packaging-motivation">
+    <h3>Motivation</h3>
+    <p>
+      There are multiple reasons to want to identify groups of SHACL Elements and to be able to know the relations between groups and individual Elements.
+      These reasons include wanting to understand the purpose of groups, how changes in one group affect other groups, and how to create new groups.
+      Three Use Cases have been articulated for this:
+    </p>
+    <ol>
+      <li>Identify groups of SHACL Elements</li>
+      <li>Find all the SHACL Elements that a group depends on</li>
+      <li>Find all the groups of SHACL Elements that a group depends on</li>
+    </ol>
+    <div style="max-height:50px; overflow:hidden;">
+      <div class="issue" data-number="624" ></div>
+    </div>
+    <div style="max-height:50px; overflow:hidden;">
+      <div class="issue" data-number="622" ></div>
+    </div>
+    <div style="max-height:50px; overflow:hidden;">
+      <div class="issue" data-number="623" ></div>
+    </div>
+  </section>
+  <section id="packaging-recommendations">
+    <h3>Recommendations Summary</h3>
+    <p>
+      Solutions to the Use Cases indicated above result in the following recommendations that build on well-established practice, such as the use of instances of <code>owl:Ontology</code> to group SHACL Elements:
+    </p>
+    <ol>
+      <li><strong>Instances of <code>owl:Ontology</code> SHOULD be used to group instance of SHACL Elements</strong></li>
+      <li><strong>An instance of a SHACL Element SHOULD indicate the instance of an <code>owl:Ontology</code> that defines it with the <code>rdfs:isDefinedBy</code> predicate</strong></li>
+      <li><strong>An instance of a SHACL Element SHOULD indicate instances of <code>owl:Ontology</code> that use it, but do not define it, with the <code>rdfs:member</code> predicate</strong></li>
```

Comment:
Isn't this backwards? The domain of `rdfs:member` is the collection and the range are the values, so shouldn't this be something like:
```suggestion
      <li><strong>An instance of <code>owl:Ontology</code> that uses a SHACL Element, but does not define it, SHOULD indicate the SHACL Element as a member with the <code>rdfs:member</code> predicate</strong></li>
```